exports/exports_test.go

```diff
@@ -0,0 +1,218 @@
+package exports_test
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+
+	"github.com/go-chi/chi/v5"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/redhatinsights/platform-go-middlewares/identity"
+
+	"github.com/redhatinsights/export-service-go/config"
+	"github.com/redhatinsights/export-service-go/exports"
+	"github.com/redhatinsights/export-service-go/logger"
+	emiddleware "github.com/redhatinsights/export-service-go/middleware"
+	"github.com/redhatinsights/export-service-go/models"
+	es3 "github.com/redhatinsights/export-service-go/s3"
+)
+
+func GenerateExportRequestBody(name, format, sources string) (exportRequest []byte) {
+	return []byte(fmt.Sprintf(`{"name": "%s", "format": "%s", "sources": [%s]}`, name, format, sources))
+}
+
+func CreateExportRequest(name, format, sources string) *http.Request {
```

Comment:
Nit-picky:  Does this method need to be exposed outside of the exports_test package?

Reply:
Ah!  I didn't notice the re-use.  The tests are in the same `exports_test` package so it should be possible to share the the function between the 2 test files without having the expose it.

Btw, its not a big deal to expose this.  I just dont like to expose anymore that is required/necessary.